libkpod/containerData.go

```diff
@@ -62,18 +60,28 @@ func GetContainerData(store storage.Store, name string, size bool) (*ContainerDa
 	if err != nil {
 		return nil, errors.Wrapf(err, "error reading build container %q", name)
 	}
-	cid, err := libkpodimage.GetContainerCopyData(store, name)
+	container, err := store.Container(name)
 	if err != nil {
-		return nil, errors.Wrapf(err, "error reading container image data")
+		return nil, errors.Wrapf(err, "error reading container data")
 	}
+
+	m := getBlankSpec()
 	config, err := store.FromContainerDirectory(ctr.ID(), "config.json")
-	if err != nil {
+	if err != nil && !os.IsNotExist(errors.Cause(err)) {
```

Comment:
If cri-o has never started the container, that file won't exist.  This lets us keep going, for example, with buildah containers.